deployments/helm/templates/_helpers.tpl

```diff
@@ -44,6 +44,10 @@ Set IP Family
 {{- printf "%s.%s.%s" .Values.vlan.networkServiceName .Values.trench.name .Release.Namespace -}}
 {{- end -}}
 
+{{- define "meridio.vlan.extInterfaceName" -}}
+{{- printf "ext-vlan.%d" .Values.vlan.id -}}
```

Comment:
I get this error with Helm 3.7.0:
```
2022-02-10T13:15:08.797528164Z stderr F Feb 10 13:15:08.797[31m [FATA] [cmd:[/bin/app]] [0merror processing rootConf from env: envconfig.Process: assigning NSM_NETWORK_SERVICES to NetworkServices: converting 'kernel://external-vlan.trench-a.red/ext-vlan.%!s(float64=100)' to type []url.URL. details: parse "kernel://external-vlan.trench-a.red/ext-vlan.%!s(float64=100)": invalid URL escape "%!s"
```
To fix it, replace `.Values.vlan.id` with `(.Values.vlan.id | int)`